src/olympia/addons/utils.py

```diff
@@ -247,3 +259,130 @@ def get_error_message(self):
         except IndexError:
             msg = None
         return msg
+
+
+class PermissionEnablerCreator:
+    """Helper class to create new permission enabler add-ons and versions.
+
+    Assumes parameters have already been validated beforehand."""
+
+    SITE_PERMISSIONS = {
+        'webmidi': 'WebMIDI',
+    }
+    MIN_VERSION = '96.0'
```

Comment:
I think we should move these to constants (a specific file maybe), if they're going to grow to a longer list of permissions.

Reply:
Yep, I'll do that. And the min version needs to be 97 actually.